src/components/varsom-weather-observation/varsom-weather-observation.tsx

```diff
@@ -22,7 +22,43 @@ export class VarsomWeatherObservation {
   @Prop() Comment: any;
   @Prop() Attachments?: Attachment[];
   
-  
+   get WindInfo():string{
+    return this.WindSpeed + " m/s " + this.strings.Observations.WeatherObservation.WindDirection + " " + this.WindDirectionInfo;
```

Comment:
Vet ikke om du vet om det, men du kan bruke "[template literal types](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html)" i Typescript tilsvarende som i C# - gjør det lettere å lese kode hvor man setter sammen tekster.
Da ville det blitt sånn, tror jeg:
`return `${this.WindSpeed} m/s ${this.strings.Observations.WeatherObservation.WindDirection} ${this.WindDirectionInfo}`;`

Reply:
Takk for tipset! Er egentlig klar over det, men hadde glemt det. Er jo litt mer elegant :)